README.md

```diff
@@ -40,31 +40,28 @@ The class components in the current version are `NumberGuessingGame` and `GuessC
 If you want to attempt this on your own without a step by step walkthrough first then leave the section below collapsed.
 
 ## Step by Step
-<details>
-    <summary>Expand to see step by step actions to convert to function components with hooks.</summary>
+
+Below are step by step actions to convert to function components with hooks.
 
 ### Convert `GuessControl`
 
-- [ ] Open `src\GuessControl.js`
-- [ ] Rename the current `GuessControl` class to `GuessControlOld` if you want to keep it a reference while converting the code
-- [ ] Create a new function component called `GuessControl`
+- [ ] Open `/src/GuessControl.js`
+- [ ] Rename the current `GuessControl` class to `GuessControlOld` if you want to keep it a reference while converting the code.
+- [ ] Create a new function component called `GuessControl` with parameter `props`.
 - [ ] Copy the return value from the render function in the class component to be the return value in the new function component. Remove any references to `this.` since those will be replaced with new references.
-- [ ] Create `currentGuess` and `setCurrentGuess` state variables using the `useState` hook and initialize the value to an empty string. Set the `value` property for the input element to refer to this state value. (Make sure you include `useState` as an import). 
-- [ ] Create a `handleInputChange` function within the component that updates the `currentGuess` state value when the user change the value in the input. Set the `onChange` property for the input element to refer to this function
+- [ ] Create new state variable named `currentGuess` with setter `setCurrentGuess` and default value of an empty string. Set the `value` property for the input element to refer to this state value. (Make sure to import `useState`)
```

Comment:
This language was confusing to me, I replaced with the standard language I use in other assignments